pyiron_gpl/pacemaker/pacemaker.py

```diff
@@ -0,0 +1,231 @@
+# coding: utf-8
+# Copyright (c) ICAMS, Ruhr University Bochum, 2021
+# Distributed under the terms of "GPLv3", see the LICENSE file.
+
+import logging
+import numpy as np
+import os
+import pandas as pd
+import re
+import ruamel.yaml as yaml
+from pyiron_base import GenericJob, GenericParameters
+from pyiron_base.settings.generic import Settings
+from shutil import copyfile
+
+
+s = Settings()
+
+try:
+    from pyace import BBasisConfiguration, ACEBBasisSet
+
+    HAS_PYACE = True
+except ImportError as e:
+    print("Could not import `pyace` package. The package should be installed for proper functionality")
+    HAS_PYACE = False
+
+# set loggers
+loggers = [logging.getLogger(name) for name in logging.root.manager.loggerDict]
```

Comment:
I think pyiron might already have a logger set up you can hook into? Not sure though, @jan-janssen could probably quickly answer that.

Reply:
This moved up, into import section wit ha comment: # set loggers level for WARNING to avoid extra print-outs  because pyace do its own logging settings